src/main/resources/application.properties

```diff
@@ -1,9 +1,22 @@
-logging.level.org.springframework.web=DEBUG
+server.port=8080
```

Comment:
@venividiviciregina server.port default value is 8080, so it is not really required